README.md

```diff
@@ -26,6 +26,27 @@ Requires Python 3.7 or greater.
 | [sbix](https://docs.microsoft.com/en-us/typography/opentype/spec/sbix) | Yes | Only for Mac Safari due to https://github.com/harfbuzz/harfbuzz/issues/2679#issuecomment-1021419864. Only square bitmaps. Uses [`resvg`](https://github.com/RazrFalcon/resvg).|
 | [CBDT](https://docs.microsoft.com/en-us/typography/opentype/spec/cbdt) | Yes |  Only square bitmaps. Uses [`resvg`](https://github.com/RazrFalcon/resvg).|
 
+### Adding color tables to existing fonts
+
+:warning: _under active development, doubtless full of bugs_
+
+Given at least one vector color table (COLR or SVG) the other vector color table and bitmap table(s)
+can be generated:
+
+```shell
+# Adds COLR to a font with SVG and vice versa
+maxmium_color my_colr_font.ttf
+```
+
+The intended result is a font that will Just Work in any modern browser:
```

Comment:
Oh good point. @drott is there any chance we could fix that?

Reply:
Added a note to readme wrt chrome preferences

Reply:
See https://bugs.chromium.org/p/skia/issues/detail?id=12945 - discussed this with Ben and it would need changes in FreeType to block it from looking at `CBDT/CBLC`.

Reply:
thanks for the update